3/debian-10/rootfs/opt/bitnami/scripts/libmoodle.sh

```diff
@@ -130,8 +134,8 @@ moodle_initialize() {
         db_name="$MOODLE_DATABASE_NAME"
         db_user="$MOODLE_DATABASE_USER"
         db_pass="$MOODLE_DATABASE_PASSWORD"
-        [[ "$db_type" = "mariadb" || "$db_type" = "mysqli" ]] && moodle_wait_for_mysql_db_connection "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass"
-
+        [[ "$db_type" = "mariadb" || "$db_type" = "mysqli" || "$db_type" = "pgsql" ]] && moodle_wait_for_db_connection "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass"
```

Comment:
I would recommend to create a new function for checking PostgreSQL connections, i.e.:

```suggestion
        [[ "$db_type" = "mariadb" || "$db_type" = "mysqli" ]] && moodle_wait_for_mysql_connection "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass"
        [[ "$db_type" = "pgsql" ]] && moodle_wait_for_postgresql_connection "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass"
```